docs/docs/1.0.6/documentation/health.md

```diff
@@ -0,0 +1,160 @@
+# Health checks
+
+Health is an extremely important part of the microservice architecture for various reasons. Load balancers rely on the 
+health status of the application while picking endpoint from load balancer set. Users want the application state to be 
+observable through the GUI or CLI. Orchestrators like Kubernetes also want to know the application status to manage 
+lifecycle of the application. 
+
+Kuma supports several aspects of the health checking. There are two policies which allows configuring active and passive:
+[Health Check](../policies/health-check.md) and [Circuit Breaker](../policies/circuit-breaker.md).
+
+Kuma is able to track the status of the Envoy proxy. If grpc stream with Envoy is disconnected then Kuma considers this 
+proxy as offline. 
```

Comment:
...but we still send the traffic regardless of that. It's not really tracking the status of Envoy itself, it is tracking connection between CP<->DP